README.md

```diff
@@ -61,26 +81,30 @@ To build on OSX, see
 for more information.
 
 ## Manual ##
-Go to https://docs.ctags.io for the preformatted documentations
-of the latest development version.
+The primary documents of Universal Ctags are man pages.
+Users should first consult the
```

Comment:
I'm afraid people using old u-ctags refers the latest documents at the web.
How about following sentences?
Users should first consult the man pages in the source tree (man/ctags.1) or a binary package you installed.
Preformatted manuals for the latest source is at [docs.crags.io](https://docs.ctags.io/en/latest/man-pages.html).

Reply:
I did not take this, because It is obvious the information on the web may be newer than one for installed version.
This sentence should be simple to let readers know man pages are primary document.  I had been misunderstanding until I discussed with you about this.